bogenliga/src/app/modules/shared/components/tables/control/base-table-sorter.class.ts

```diff
@@ -60,7 +60,10 @@ export abstract class BaseTableSorter {
       console.warn('No table configuration found. Abort sorting column: ', sortColumn.propertyName);
       return rows;
     }
-
+    // the columns are not sortable --> return rows without sorting
+    if(!this.config.columns[this.config.columns.length - 1].sortable && !this.config.columns[0].sortable){
+      return rows;
+    }
```

Comment:
Da hat sich doch noch ein Fehler eingeschlichen. Nicht nur überprüfen ob die letzte und erste Spalte nicht sortierbar sind, sondern alle.
Geht z.B. so: 
`if (this.config.columns.filter((c) => c.sortable).length === 0) {
      return rows;
    }`

Reply:
Ansonsten wäre die Tabelle wieder sortierbar, wenn man z.B. die zweite Spalte auf "sortable = true" macht